flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```

Comment:
about this:

![image](https://user-images.githubusercontent.com/114441/173977068-474151eb-3366-4a39-bc20-fe65bdd126eb.png)


tl;dr please add aarch64 support.

Reply:
and if this contradicts https://github.com/flathub/flathub/pull/3222/files#r884392645 it's because I assumed you had checked.

Reply:
@hfiguiere I added a simple update for multiarch support [here](https://github.com/CleoMenezesJr/flathub/pull/2), but I don't have an aarch64 machine to test it on. I'll try it in a virtual machine when I get the chance, but it would be nice to have someone verify that this works (or why it doesn't) on real hardware).

Reply:
then why is it not here? the best way to test is to start by building it.

Reply:
Now it's here, now wait for the build

Reply:
@hfiguiere It really was something that went unnoticed for me. My bad